packages/fcl/src/current-user/exec-service/strategies/http-post.js

```diff
@@ -26,7 +25,7 @@ export async function execHttpPost(service, signable, opts) {
     return poll(resp.updates, () => canContinue)
       .then(compositeSignature => {
         closeFrame()
-        return normalizeCompositeSignature(compositeSignature)
+        return compositeSignature
```

Comment:
can we change `compositeSiganture` to be something like `responseFromService` or `serviceResponse` or something like that, as the expectation and the need here is that it wont always be a `compositeSignature` now

Reply:
yeah, good call